tasmota/xdrv_17_rcswitch.ino

```diff
@@ -27,24 +23,26 @@
 #define D_JSON_RF_BITS "Bits"
 #define D_JSON_RF_DATA "Data"
 
-#define D_CMND_RFSEND "RFSend"
+#define D_CMND_RFSEND "RfSend"
 #define D_CMND_RFPROTOCOL "RfProtocol"
+#define D_CMND_RFTIMEOUT "RfTimeOut"
 
 #define D_JSON_RF_PULSE "Pulse"
 #define D_JSON_RF_REPEAT "Repeat"
 #define D_JSON_NONE_ENABLED "None Enabled"
 
 const char kRfCommands[] PROGMEM = "|"  // No prefix
-  D_CMND_RFSEND "|" D_CMND_RFPROTOCOL;
+  D_CMND_RFSEND "|" D_CMND_RFPROTOCOL "|" D_CMND_RFTIMEOUT;
 
 void (* const RfCommands[])(void) PROGMEM = {
-  &CmndRfSend, &CmndRfProtocol };
+  &CmndRfSend, &CmndRfProtocol, &CmndRfTimeOut};
 
 #include <RCSwitch.h>
 
 RCSwitch mySwitch = RCSwitch();
 
-#define RF_TIME_AVOID_DUPLICATE 1000  // Milliseconds
+int RF_TIME_AVOID_DUPLICATE = 3000;  // Use variable to change "RfTimeOut" to get new data from RfReceive
```

Comment:
Do not change the default value.
3000 may be the one that suits your specific use-case but in many other case this will break user's expectation as it will prevent users to send the same command within a short time. For exemple for a dimmer or a sound volume you need to press the key multiple time. Waiting 3 seconds between each press is not acceptable.

Also do not keep the name as fully capitalized. `RfTimeToAvoidDupplicate` would be better for a variable. Captalized should be only for preprocessors macros/constants.

Reply:
Corrected